gapis/api/vulkan/externs.go

```diff
@@ -1115,3 +1117,24 @@ func bindSparse(ctx context.Context, s *api.GlobalState, binds *QueuedSparseBind
 		}
 	}
 }
+
+func (e externs) onVkError(err interface{}) {
+	var issue replay.Issue
+	issue.Command = e.cmdID
+	issue.Severity = service.Severity_ErrorLevel
+	switch err := err.(type) {
+	case *ERR_INVALID_HANDLE:
+		issue.Error = fmt.Errorf("Invalid %s: %v", err.HandleType, err.Handle)
+	case *ERR_NULL_POINTER:
+		issue.Error = fmt.Errorf("Null pointer of %s", err.PointerType)
+	case *ERR_UNSUPPORTED_EXTENSION:
+		issue.Error = fmt.Errorf("Unsupported extension: %s", err.Name)
```

Comment:
I would probably make this a warning.
To me ERR_UNSUPPORTED_EXTENSION would be if you use an extension that the driver does not provide.

In THIS case we are talking about an extension that GAPID does not understand. Which is not an issue with the application itself.